WordPress/src/main/java/org/wordpress/android/ui/posts/EditorLauncher.kt

```diff
@@ -146,13 +140,14 @@ class EditorLauncher @Inject constructor(
         )
     }
 
+    private fun getFeatureFlagsString(featureState: GutenbergKitFeatureChecker.FeatureState): String {
+        return "(experimental_block_editor: ${featureState.isExperimentalBlockEditorEnabled}, " +
+                "gutenberg_kit_feature: ${featureState.isGutenbergKitFeatureEnabled}, " +
+                "disable_experimental_block_editor: ${featureState.isDisableExperimentalBlockEditorEnabled})"
+    }
+
     private fun getFeatureFlagsString(): String {
```

Comment:
We can probably get rid of this one with a default parameter.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/22247.